Prefetching-Library/android_prefetching_lib/src/main/java/nl/vu/cs/s2group/nappa/prefetch/GreedyPrefetchingStrategy.java

```diff
@@ -33,7 +34,7 @@
  * of node candidates generated, a threshold value is inserted.
  */
 public class GreedyPrefetchingStrategy implements PrefetchingStrategy {
-    private final static String LOG_TAG = GreedyPrefetchingStrategy.class.getSimpleName();
+    private static final String LOG_TAG = GreedyPrefetchingStrategy.class.getSimpleName();
```

Comment:
While it is not an error, it seems like writing static before final throws a code smell warning. Since it was a simple matter of find/replace and they all occur in the files modified in this PR, I added this change for this PR